### DISCUSSION TITLE
Header dropdown menu position is not relevant to the Header button

### DISCUSSION THREAD

**Discussion by danleydmello** [iTwin/iTwinUI#84] [Q&A] [answered]
I already discussed this with @veekeys , and it seems that it needs js to position the menu to  be responsive. But I was wondering for header we can have special case without needing a js to position the menu as header is stationary at the top of the UI.

**Comment by mayank99** [comment, marked as answer]
Not sure I understand the problem here.

If you're talking about the React implementation, Header button relies on our DropdownMenu component, which is implemented using JS. Sure it's a simple case that could be done with pure CSS, but we want to reuse our existing components wherever possible.

You are not forced to use JS in your implementation (we don't have any JS in the base CSS package). So you could write your own rules/markup for handling the menu. Using `:focus-within` is one approach of doing it.

**Reply by veekeys** [reply]
What @danleydmello is suggesting I think, is that maybe we could check up to provide default positioning only from css as Header is not a component with dynamic positioning. So theoretically, we could be able to set positioning from css part.
I guess it not high priority, but an idea we could check up at some time.
